### DISCUSSION TITLE
Error when uploading a csv file saying I need to upload a csv file?

### DISCUSSION THREAD

**Discussion by Martin-QPT** [AntlerVC/firetable#901] [Support / Q&A] [answered]
The error is below. I have no idea why I'd get this error.
![image](https://user-images.githubusercontent.com/108190238/198175300-424df653-6ab8-45ef-9955-e45649fed251.png)

**Comment by BeeBombshell** [comment, marked as answer]
Hi @Martin-QPT, I think this issue is regarding the format in which the file is saved. Try importing your data to a spreadsheet using Excel or similar software. Then, save it in the `.csv` format and see if the problem still persists.

<img width="272" alt="image" src="https://user-images.githubusercontent.com/53828745/198232411-7ab2c4c3-3212-49a0-8801-942ce3fc42c4.png">

As you can see, while saving an Excel spreadsheet the file format differs when compared to saving a Notepad document. Let me know if you're still facing the issue after giving this a go.

**Comment by harinij** [comment]
@Martin-QPT you can also check the format of the file contents to make sure it is in valid csv format. If you can share a small sample `csv` file that you are uploading here, we can verify it as well. When you upload a valid csv or tsv file you will see a successful check like the below screen and you will be able to continue.
<img width="504" alt="Screen Shot 2022-11-02 at 2 55 33 pm" src="https://user-images.githubusercontent.com/307298/199392270-1c65b6dd-bce7-4094-a044-d8a13ee7ea6d.png">
